### DISCUSSION TITLE
How to rearrange document fields?

### DISCUSSION THREAD

**Discussion by zanami** [keystonejs/keystone#7883] [Questions]
I'm looking at [Document Fields Demo](https://keystonejs.com/docs/guides/document-field-demo) and can't figure out how to move the field "blocks" around? I also could not insert a new "block" between existing ones.
Am I missing something?

**Comment by flexdinesh** [comment]
> re: rearranging blocks

It's not supported at the moment and we'll take this into consideration for our roadmap

> re: insert new block between blocks

You're not missing anything. Unfortunately this is a known issue at the moment. This happens when blocks are wrapped with `NonEditable` or don't have editable `fields.child`. We'll update soon once we have a fix.
